tools/clang/test/CodeGenHLSL/short_circuiting0.hlsl

```diff
@@ -0,0 +1,14 @@
+// RUN: %dxc -E main -T ps_6_0 %s | FileCheck %s
+
+// inc(color) will add 1 to color.
+// Make sure result is add 5.2.
+// CHECK: 0x4014CCCCC0000000
+
+bool inc(inout float4 v) { v++; return true; }
+
+float4 main(int val: A, float4 color: COLOR) : SV_TARGET {
```

Comment:
Sorry for my previous examples. Actually `int val: A` is not used here and can be deleted to make the test case more concise and to the point. :) Same for the other cases.